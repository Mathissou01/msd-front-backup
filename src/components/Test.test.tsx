import { render, screen } from "@testing-library/react";

test("renders learn test", () => {
  render(<div>testing?</div>);
  const linkElement = screen.getByText("testing?");
  expect(linkElement).toBeInTheDocument();
  expect(linkElement).toMatchSnapshot();
});
