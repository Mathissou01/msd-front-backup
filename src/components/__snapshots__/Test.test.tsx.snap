// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn test 1`] = `
<div>
  testing?
</div>
`;
